dm/dm.go

```diff
@@ -71,13 +72,15 @@ var usage = func() {
 	fmt.Fprintln(os.Stderr, "Flags:")
 	flag.PrintDefaults()
 	fmt.Fprintln(os.Stderr)
-	os.Exit(1)
+	fmt.Fprintln(os.Stderr, "--stdin requires a tar archive of input files and a top level file name.")
```

Comment:
I admit it's been a looong time, but does Windows come with tar? Can we look at the file and see if it's a tar file and if not, treat it as a single file?